charts/argo-cd/templates/redis/deployment.yaml

```diff
@@ -139,8 +189,12 @@ spec:
         {{- end }}
         {{- end }}
       {{- end }}
-      {{- with .Values.redis.volumes }}
       volumes:
+        - name: health
+          configMap:
+            name: {{ include "argo-cd.redis.fullname" . }}-health-configmap
+            defaultMode: 0755
```

Comment:
This ConfigMap is created when `if and .Values.redis.enabled (not $redisHa.enabled)` , so we also need same condition to refer here ...? 👀

Reply:
The whole deployment.yaml file is enclosed in that if , see [line 2](https://github.com/argoproj/argo-helm/pull/2400/files/8497df2d7795fe948c960b6cc0cb666138a379e8#diff-cc2af74d39b4cb33e7fe369cddbed518c6a636083bdaee5e803a142584b918d1L2)

Reply:
Oh thank you!